src/sfizz/Synth.cpp

```diff
@@ -1343,6 +1344,26 @@ void Synth::hdAftertouch(int delay, float normAftertouch) noexcept
     impl.performHdcc(delay, ExtendedCCs::channelAftertouch, normAftertouch, false);
 }
 
+void Synth::polyAftertouch(int delay, int noteNumber, uint8_t aftertouch) noexcept
+{
+    const float normalizedAftertouch = normalize7Bits(aftertouch);
+    hdPolyAftertouch(delay, noteNumber, normalizedAftertouch);
+}
+
+void Synth::hdPolyAftertouch(int delay, int noteNumber, float normAftertouch) noexcept
+{
+    Impl& impl = *impl_;
+    ScopedTiming logger { impl.dispatchDuration_, ScopedTiming::Operation::addToDuration };
+
+    impl.resources_.midiState.polyAftertouchEvent(delay, noteNumber, normAftertouch);
+
+    for (auto& voice : impl.voiceManager_)
+        voice.registerPolyAftertouch(delay, noteNumber, normAftertouch);
+
+    // Note information is lost on this CC
```

Comment:
>    // Note information is lost on this CC

What is the reason for not keeping it?

Reply:
Well there is note + value on the polyphonic aftertouch event, and only value in the CC event, so we cannot "translate" it entirely.

It's only to start regions on polyphonic aftertouch using `on_lo/hicc130` and activate regions using `lo/hicc130` though, envelopes like e.g. `pan_cc130` will be per-voice and use the note information---see the polyphonicAftertouch section in `Controller.cpp`.